plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotPageSourceProvider.java

```diff
@@ -75,7 +78,7 @@ public ConnectorPageSource createPageSource(
         }
         PinotTableHandle pinotTableHandle = (PinotTableHandle) tableHandle;
         String query = generatePql(pinotTableHandle, handles, pinotSplit.getSuffix(), pinotSplit.getTimePredicate(), limitForSegmentQueries);
-
+        LOG.debug("Pinot query: %s", query);
```

Comment:
The logging place looks still wrong. Note that #22253 is going to move the above `generatePql` method call.